phoebe/atmospheres/passbands.py

```diff
@@ -3084,22 +2986,22 @@ def Inorm_bol_bb(Teff=5772., logg=4.43, abun=0.0, atm='blackbody', intens_weight
     )
 
     pb.compute_blackbody_intensities(include_extinction=True)
-#    pb.compute_bb_reddening(verbose=True)
+    # pb.compute_bb_reddening(verbose=True)
```

Comment:
I think we should just go ahead and delete these references to the old extinction calculations now these functions don't exist.

Reply:
Agreed, done.